PWGCF/FemtoDream/femtoDreamPairTaskTrackTrack.cxx

```diff
@@ -232,25 +227,28 @@ struct femtoDreamPairTaskTrackTrack {
   void processSameEvent(o2::aod::FemtoDreamCollision& col,
                         o2::aod::FemtoDreamParticles& parts)
   {
+    auto groupPartsOne = partsOne->sliceByCached(aod::femtodreamparticle::femtoDreamCollisionId, col.globalIndex());
+    auto groupPartsTwo = partsTwo->sliceByCached(aod::femtodreamparticle::femtoDreamCollisionId, col.globalIndex());
+
     const int multCol = col.multV0M();
     eventHisto.fillQA(col);
     /// Histogramming same event
-    for (auto& part : partsOne) {
+    for (auto& part : groupPartsOne) {
       if (!isFullPIDSelected(part.pidcut(), part.p(), cfgCutTable->get("PartOne", "PIDthr"), vPIDPartOne, cfgCutTable->get("PartOne", "nSigmaTPC"), cfgCutTable->get("PartOne", "nSigmaTPCTOF"))) {
         continue;
       }
       trackHistoPartOne.fillQA(part);
     }
     if (!ConfIsSame) {
```

Comment:
Does ConfIsSame refer to "having particles of the same kind" or "taking pairs with exactly same particle repeated"?

Reply:
Particles of the same kind